guide/src/continuous-integration.md

```diff
@@ -0,0 +1,38 @@
+# Continuous integration
+
+This chapter shows some examples how to use Yarner with Continuous integration.
+There are always two ways to get Yarner:
+
+1. Use a rust image and install it using `cargo`
+2. Download the [precompiled binaries](https://github.com/mlange-42/yarner/releases/)
+
+In these examples, we use the second option.
+
+[[_TOC_]]
+
+## GitHub with Travis-CI
+
+## GitHub Actions
+
+## GitLab CI
+
+In your GitLab project, create a file `.gitlab-ci-yml` with the following content:
+
+```yml
+variables:
+  YARNER_VERSION: 0.2.2
+
+before_script:
+  - curl -L -o- https://github.com/mlange-42/yarner/releases/download/${YARNER_VERSION}/yarner-${YARNER_VERSION}-linux.tar.gz | tar xz
```

Comment:
This is not so much CI-specific but a general question. Does the archive contain anything but the Yarner binary? If not, maybe we should just drop the subfolder within the archive?

Reply:
Yes, its only the executable. I did it this way because I don't like seeing files tumbling at me when extracting an archive, and normally expect things to be within a folder. Also, in the current state it is more convenient for simly moving the folder to tha programs folder. So I am not entirely convinced yet.

Reply:
Fair enough.